docker-compose.yml

```
@@ -1,106 +1,205 @@
+              capabilities: [gpu]
 
-  video:
-    build: ./video
+  lookup:
+    build: ./lookup
+    ports:
+      - "5000:5000"
     volumes:
-    - ./crawler/crawled_faces:/app/crawled_faces:ro
-    - ./crawler/transcriptions:/app/transcriptions
+      - ./lookup:/app # Keeps source code in sync for dev
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)